globus_sdk/transfer/client.py

```diff
@@ -334,6 +334,67 @@ def submit_delete(self, data):
         """
         return self.post('/delete', data)
 
+    def make_submit_transfer_data(self, source_endpoint, dest_endpoint,
+                                  transfer_items, label, sync_level=None,
+                                  **kwargs):
```

Comment:
I think label should be optional in the sdk. If label is None, the webapp displays the source/dest endpoint names, which is probably good enough for a lot of common use cases.